### ISSUE TITLE
Use local storage with non-string types

### ISSUE THREAD

**Issue by jasonaowen** [mjec/rc-niceties#31]
The front-end React app uses browser local storage for several things, including to ensure that in-progress niceties are not lost when the window is closed. However, the [localStorage.setItem](https://developer.mozilla.org/en-US/docs/Web/API/Storage/setItem) API only works with strings. This has led us to [storing boolean values as strings](https://github.com/mjec/rc-niceties/blob/b75d69fc55bd99d04ce3a0557798b1fb249bd0f6/src/App.js#L16), which is both confusing to developers and can lead to bugs if those values are passed to a back-end API that expects actual JSON boolean values.

This limitation of the browser API is not at all unique to this project, and I'm certain that several people have solved it in a reusable way. Find a dependency that abstracts away this issue, and convert our code to use it instead of directly using the local storage API.

**Comment by jasonaowen** [comment]
Credit to @christalee for raising this!

A [quick search](https://www.npmjs.com/search?q=localstorage%20json) revealed several options, the most popular of which appears to be [store2](https://www.npmjs.com/package/store2). I haven't used it and this is not a recommendation; merely a suggestion of where to start research.

**Comment by christalee** [comment]
I used store2 for lack of a better-looking alternative. I stuck with `store.get(key)/store.set(key, value)` instead of `store(key)/store(key, value)` for clarity. (This change was prioritized b/c current versions of SQLAlchemy don't accept string booleans.)

I tested this by simply clearing LocalStorage and confirming that the data repopulated properly, and checking that no errors were thrown. I'll hold off on submitting the change until PR #35 lands, to avoid confusion.
